fv3core/fv3core/stencils/saturation_adjustment.py

```diff
@@ -577,6 +586,43 @@ def satadjust(
     fac_l2v: float,
     last_step: bool,
 ):
+    """
+    Args:
+        peln (in):
```

Comment:
Looks like peln is only used if the model is hydrostatic, can we add a TODO to delete the hydrostatic parts of this code? It will increase memory performance.